sale_order_general_discount/models/sale_order_line.py

```diff
@@ -1,11 +1,33 @@
 # Copyright 2018 Tecnativa - Sergio Teruel
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
-from odoo import api, models
+from odoo import api, fields, models
 
 
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
+    discount = fields.Float(
+        compute="_compute_discount",
+        store=True,
+        readonly=False,
+    )
+
+    @api.model_create_multi
+    def create(self, vals_list):
```

Comment:
I don't like this override, and doing the field computed writable shouldn't be needed.

Reply:
I have check funcionality after I remove this override and work fine.